<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="data_dir"          default="/mnt/md0/gear_data"/>
  <arg name="image_def"         default="hd"/>
  <arg name="image_prefix"      default="im"/>      

  <!-- Playback node -->
  <node pkg="gear_data_handler" type="playback" name="playback" output="screen">
    <param name="data_dir"          value="$(arg data_dir)"/>
    <param name="image_def"         value="$(arg image_def)"/>
    <param name="image_prefix"      value="$(arg image_prefix)"/>
  </node>
</launch>